week01/TwoSum.cpp

```diff
@@ -0,0 +1,18 @@
+class Solution{
+public:
+    vector<int> twoSum(vector<int>& nums, int target)
+    {
+        unordered_map <int, int> mp;
+        vector <int> ans;
+        mp[ nums[0] ] = 1;
+        for(int i = 1; i < nums.size(); i++){
+            if(mp[ target - nums[i] ]){
```

Comment:
This is a dangerous operation! A serious memory leak is happening here. You should ALWAYS use `mp.count(item)` to check the existence of an element. It is a very trivial knowledge that we use in Competitive Programming also.